2024-06-17/main.py

```diff
@@ -0,0 +1,201 @@
+import unittest
+import sqlite3
+import uuid
+from pprint import pprint
+from fastapi import FastAPI, APIRouter, HTTPException
+
+# Create our in-memory DB and cursor
+db = sqlite3.connect("file::memory:?cache=shared", check_same_thread=False)
+cur = db.cursor()
+
+# Create your table SQL statement
+def createDB():
+    cur.execute("CREATE TABLE users(id, name, address, email, phone)")
```

Comment:
So the statement on 8 creates the database -- this is just adding a table?

Reply:
right?...